File: khalti-android/src/main/java/com/khalti/checkOut/Wallet/helper/WalletInitPojo.java
package com.khalti.checkOut.Wallet.helper;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WalletInitPojo {
    @SerializedName("token")
    @Expose
    private String token;
    @SerializedName("pin_created")
    @Expose
    private boolean pinCreated;
    @SerializedName("pin_created_message")
    @Expose
    private String pinCreatedMessage;

    public String getToken() {
        return token;
    }

    public boolean isPinCreated() {
        return pinCreated;
    }

    public String getPinCreatedMessage() {
        return pinCreatedMessage;
    }
}

File: khalti-android/src/main/java/com/khalti/checkout/form/helper/WalletInitPojo.kt
package com.khalti.checkout.form.helper


import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class WalletInitPojo {
    @SerializedName("token")
    @Expose
    val token: String? = null
    @SerializedName("pin_created")
    @Expose
    val isPinCreated: Boolean = false
    @SerializedName("pin_created_message")
    @Expose
    val pinCreatedMessage: String? = null
}